log/logger_test.go

```diff
@@ -44,20 +60,24 @@ func setupTestLoggers() {
 	RWM.Lock()
```

Comment:
Can't highlight it, but your new changes make the sublogger name of `TEST` invalid. You'll need to update it to fix tests